network-mux/src/Network/Mux/DeltaQTraceStats.hs

```diff
@@ -0,0 +1,258 @@
+module Network.Mux.DeltaQTraceStats
+ ( step
+ , OneWayDeltaQSample(..)
+ , constructSample
+ , StatsA
+ , initialStatsA
+ )
+where
+
+import           Data.IntMap.Strict (IntMap)
+import qualified Data.IntMap.Strict as IM
+import           Data.Maybe
+import           Data.Word (Word32)
+import           Numeric.IEEE (nan)
```

Comment:
It looks like we only use the `ieee754` package for this `nan`, but you can just use
```
nan :: Float
nan = 0/0
```

Reply:
ieee754 is already in the set of packages used by the network (it was already in the selected set) so doesn't appear to increase the package set (I did check this)

Reply:
Ok. I wonder why we already depend on it.